src/overlays/actors/ovl_Bg_Astr_Bombwall/z_bg_astr_bombwall.h

```diff
@@ -5,15 +5,16 @@
 
 struct BgAstrBombwall;
 
-typedef void (*BgAstrBombwallActionFunc)(struct BgAstrBombwall*, GlobalContext*);
+#define OBJBgAstrBombwall_GET_SWITCHFLAG(this) (this->dyna.actor.params & 0x7F)
```

Comment:
```suggestion
#define ASTRBOMBWALL_GET_SWITCHFLAG(thisx) ((thisx)->params & 0x7F)
```
Our standard is for the macro to be all CAPS, in this case that would be `OBJBGASTRBOMBWALL_GET_SWITCHFLAG`, though shortening it to `ASTRBOMBWALL_GET_SWITCHFLAG` seems better. Also the macro should take in the base actor pointer not the specific actor, along with parenthesis around the macro arg. Note this would require changing the uses of the macro to `ASTRBOMBWALL_GET_SWITCHFLAG(&this->actor)`.